server/game/cards/characters/04/aerondamphair.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../../drawcard.js');
+
+class AeronDamphair extends DrawCard {
+
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardSaved: event => event.card.getType() === 'character'
+            },
+            choices: {
```

Comment:
Using choices here is not strictly correct. While the card seems to offer a choice, the fact that abilities can only be triggered if you would change the game state dictates that after triggering you have to either kneel the card (if standing) or stand it (if kneeling)